adapters/yahoossp/yahoossp.go

```diff
@@ -156,13 +159,20 @@ func changeRequestForBidService(request *openrtb2.BidRequest, extension *openrtb
 		request.App.ID = extension.Dcn
 	}
 
-	if request.Imp[0].Banner == nil {
-		return nil
+	if request.Imp[0].Banner != nil {
+		banner := *request.Imp[0].Banner
+		request.Imp[0].Banner = &banner
+
+		err := validateBanner(&banner)
+		if err != nil {
+			return err
+		}
```

Comment:
I'm interested in why this code was updated for this PR? Does it help support video in some way, or did you just want to tweak how Banner is supported?

Reply:
Just some small internal refactoring, to move banner validation into a particular function. I think it improves the readability.